### ISSUE TITLE
Package activeState, does not support version information submission review

### ISSUE THREAD

**Issue by ccfiel** [shr3jn/fastlane-plugin-huawei_appgallery_connect#30]
The app does not automatically submit and got this error "Package activeState, does not support version information submission review".  Any idea why?

```
/root/.gem/ruby/ruby/2.7.0/gems/fastlane-plugin-huawei_appgallery_connect-1.0.16/lib/fastlane/plugin/huawei_appgallery_connect/helper/huawei_appgallery_connect_helper.rb:245: warning: URI.escape is obsolete
+------------------+---------------------+
|              Lane Context              |
+------------------+---------------------+
| DEFAULT_PLATFORM | android             |
| PLATFORM_NAME    | android             |
| LANE_NAME        | android app_gallery |
+------------------+---------------------+
[09:53:38]: {"ret"=>{"code"=>204144641, "msg"=>"[AppGalleryConnectPublishService]Package activeState, does not support version information submission review"}}
+------+---------------------------+-------------+
|                fastlane summary                |
+------+---------------------------+-------------+
| Step | Action                    | Time (in s) |
+------+---------------------------+-------------+
| 1    | default_platform          | 0           |
| 2    | default_platform          | 0           |
| 3    | pwd                       | 0           |
| 💥   | huawei_appgallery_connect | 9           |
+------+---------------------------+-------------+
[09:53:38]: fastlane finished with errors
[!] {"ret"=>{"code"=>204144641, "msg"=>"[AppGalleryConnectPublishService]Package activeState, does not support version information submission review"}}

```

**Comment by adriangl** [comment]
Hello! I also found the same error when trying to release an APK: the app is left in draft state but the action does not release it.

[There have been some changes in the Connect API lately](https://developer.huawei.com/consumer/en/doc/development/AppGallery-connect-Guides/agcapi-pub-releasenotes-0000001111685226) so it may (or it may not) be related to it. Hope it can get fixed!

**Comment by shr3jn** [comment]
Hi! I will check the issue.

**Comment by ccfiel** [comment]
@shr3jn thanks!

**Comment by shr3jn** [comment]
> Hello! I also found the same error when trying to release an APK: the app is left in draft state but the action does not release it.
> 
> [There have been some changes in the Connect API lately](https://developer.huawei.com/consumer/en/doc/development/AppGallery-connect-Guides/agcapi-pub-releasenotes-0000001111685226) so it may (or it may not) be related to it. Hope it can get fixed!

I've verified the docs and nothing has changed.

**Comment by shr3jn** [comment]
@ccfiel Can you raise a ticket on Huawei Support?

**Comment by ccfiel** [comment]
@shr3jn I did create a ticket and a issue in forum https://forums.developer.huawei.com/forumPortal/en/topic/0202741612330060582. Are still experiencing this also?

**Comment by ccfiel** [comment]
@shr3jn they reply this is the reply

```
 Dear developer,

        You have received a reply for your work order: Uploading APK using API error (D170933) , The progress is as follows:

Dear developer,


Thank you for contacting us!!

This error will come when you will be providing any wrong parameter to the request body. 

For details , please refer to: https://developer.huawei.com/consumer/en/doc/development/AppGallery-connect-References/agcapi-publishingapi-errorcode-0000001163523297

  Dear developer,

        You have received a reply for your work order: Uploading APK using API error (D170933) , The progress is as follows:

Dear developer,


Thank you for contacting us!!

This error will come when you will be providing any wrong parameter to the request body. 

For details , please refer to: https://developer.huawei.com/consumer/en/doc/development/AppGallery-connect-References/agcapi-publishingapi-errorcode-0000001163523297

 ```

**Comment by marhaupe** [comment]
We're having the same issue. I just checkout out a version in our repo that has been successfully uploaded to the AppGallery on Nov 16h, 10:29 UTC with this plugin. Running it again today results in the same error:

```ruby
{"ret"=>{"code"=>204144641, "msg"=>"[AppGalleryConnectPublishService]Package activeState, does not support version information submission review"}}
Deployment failed! 😩
```

So something changed with either this plugin or with Huawei AppGallery. I suppose it's the latter, but I admit that the changelog looks clean.

**Comment by TobiGeh** [comment]
Got the same response, out of thin air. the upload script wasn't changed.
For debugging i have created a simple http request in intelliJ:
POST .../appsubmit?appid=...
Authorization: ...
client_id: ...

Thats it and it has worked without problems. So for me it has something to do with a request out of a script

**Comment by ccfiel** [comment]
@shr3jn any updates? :)

**Comment by shr3jn** [comment]
Fixed on v1.0.17. 

The issue was caused due to instantly submitting the app for review after apk upload. Added a 10 second delay before submitting for review.

**Comment by shr3jn** [comment]
[14:18:45]: ------------------------------
[14:18:45]: --- Step: default_platform ---
[14:18:45]: ------------------------------
[14:18:45]: Driving the lane 'android huawei' 🚀
[14:18:45]: ---------------------------------------
[14:18:45]: --- Step: huawei_appgallery_connect ---
[14:18:45]: ---------------------------------------
[14:18:45]: Fetching app access token
[14:18:46]: Fetching upload URL
[14:18:47]: Uploading app
[14:18:50]: Upload app to AppGallery Connect successful
[14:18:50]: Saving app information
[14:18:56]: App information saved.
[14:18:56]: Waiting 10 seconds for upload to get processed...
[14:19:06]: Submitting app for review
[14:19:08]: Successfully submitted app for review
